app/src/main/java/org/simple/clinic/patientattribute/entry/BMIEntryEvent.kt

```diff
@@ -4,6 +4,16 @@ import org.simple.clinic.widgets.UiEvent
 
 sealed class BMIEntryEvent : UiEvent
 
+data class HeightChanged(val height: String) : BMIEntryEvent() {
+  override val analyticsName = "BMI Entry:Height Changed"
+}
+
+data class WeightChanged(val weight: String) : BMIEntryEvent() {
+  override val analyticsName = "BMI Entry:Weight Changed"
```

Comment:
These 2 analytic changes might be too much, since this gets triggered as they change the value. Just on done/save button should be fine, right?

Reply:
Done